src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+
+type Props = {
+  isHidden: boolean;
+  setIsHidden: (arg: boolean) => void;
+  onError: string;
```

Comment:
1. `onError` sounds like a callback function name, I would recommend to name it as 'errorType'
2. If you know all posiible values of prop - create new type or enum of it. Example:

```
// for example in some your '...types.ts' file 
export type ErrorType = 'add' | 'load' | 'delete' | 'update'
```

and then 

```
import { ErrorType  } from '...types.ts';

type Props = {
  isHidden: boolean;
  setIsHidden: (arg: boolean) => void;
  onError: ErrorType | ''
}  

```